app/scripts/modules/core/src/function/Function.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { UISref, UISrefActive } from '@uirouter/react';
+
+import { Application } from 'core/application/application.model';
+import { IFunction } from 'core/domain';
+
+import { EntityNotifications } from 'core/entityTag/notifications/EntityNotifications';
+
+export interface IFunctionProps {
+  application: Application;
+  functionDef: IFunction;
+}
+
+export class Function extends React.Component<IFunctionProps> {
```

Comment:
Can we make this a functional component since it is stateless and presentational only?